torch_np/_ndarray.py

```diff
@@ -404,6 +404,9 @@ def __setitem__(self, index, value):
         value = _helpers.ndarrays_to_tensors(value)
         return self.tensor.__setitem__(index, value)
 
+    def take(*a, **kw):
+        raise NotImplementedError()
```

Comment:
Yeah, ok for now.

This will get easier when https://github.com/Quansight-Labs/numpy_pytorch_interop/pull/94 gets in. Then `ndarray.take` will be just `take = _funcs.take` in the ndarray class body.

For historic reasons, `_funcs.py` was cleaner, circular import free module and `_wrapper.py` carried its messy history.